src/components/forms/formValidation.js

```diff
@@ -30,13 +31,19 @@ export const dateValidator = {
   msg: 'Please enter a date (YYYY-MM-DD), e.g. 2018-11-13',
 };
 
+export const dayJSValidator = {
+  id: 'dayjs',
+  isValid: (val) => isValidDayjsDate(val),
+  msg: 'Please select a valid date.',
```

Comment:
Wondering about if there is a way to make this error message more helpful, if it is a pop up calendar, what is the case where a user selects an invalid date?

Reply:
Great question!  The control includes a text field that users can type into.  The control does a good job at guarding against invalid dates, but they are still theoretically possible.  For example, you can type in 2/31/2025.